components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -252,4 +258,9 @@ class CustomTabsToolbarFeature(
             }
         }
     }
+
+    companion object {
+        private const val ACTION_BUTTON_WIDTH_DP = 24
```

Comment:
@mcarare should the minimum [touch target size be 48dp](https://support.google.com/accessibility/android/answer/7101858?hl=en)?

Reply:
This is just the size of the drawable. The view still has the same dimensions. I will change the naming to be more clear.

Reply:
Thanks for the clarification 👍

Reply:
Thanks for the review!